src/packages/admin-ui-components/src/utils/export-csv.ts

```diff
@@ -1,13 +1,19 @@
 import Papa from 'papaparse';
 
 export const convertObjectValueToString = <T>(source: T[]) => {
+	const getObjectValue = (item: any) =>
+		item.hasOwnProperty('label') ? (item as any)['label'] : JSON.stringify(item);
```

Comment:
is the type cast " (item as any)" necessary? If so all good

Reply:
done. I have removed the unnecessary code.